openfe/protocols/openmm_utils/multistate_analysis.py

```diff
@@ -125,8 +125,9 @@ def _analyze(self, forward_reverse_samples: int):
         """
         # Do things that get badly cached later
         self._replica_states = self.analyzer.reporter.read_replica_thermodynamic_states()
-        self._equil_iters = self.analyzer.n_equilibration_iterations
-        self._prod_iters = self.analyzer._equilibration_data[2]
+        # float conversions to avoid having to deal with numpy dtype serialization
+        self._equil_iters = float(self.analyzer.n_equilibration_iterations)
```

Comment:
should these be `int` instead of `float`? (Variable name looks to me like an `int`)

Reply:
I've already raised that issue in openmmtools, currently they return float and it's dumb.